test/runtests.jl

```diff
@@ -0,0 +1,30 @@
+using PkgDeps
+using Test# Create a tmp directory
```

Comment:
should there be an extra line here? 
```suggestion
using Test
# Create a tmp directory
```

Reply:
Oopsies, that shouldn't even be in there. Just an artifact from making things.